runtime/src/server/middleware/types.ts

```diff
@@ -60,4 +60,16 @@ interface Component {
 		css: { code: string, map: any };
 		html: string
 	}
-}
\ No newline at end of file
+}
+
+export interface PageContent {
```

Comment:
Not sure about sending the page context. My feeling is that it makes it more complex than it need to be.

Is it useful to be able to replace something in the `head`, that cannot be done by replacing `%sapper.head%` first?

Reply:
Page context removed in dbb6e92.